### PR TITLE
Add deploy to wp.org GitHub actions

### PR DESCRIPTION
**Pull request by pedro-mendonca**
Fix issue #12

### CODE REVIEW

File: .github/workflows/deploy-asset-readme-update.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2.4.0
+
+    - name: WordPress.org Plugin Readme/Assets Update
+      uses: 10up/action-wordpress-plugin-asset-update@2.0.0
+      env:
+        SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+        SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+        SLUG: wpt-customm-mo-file
```
Comment (fxbenard) [inline .github/workflows/deploy-asset-readme-update.yml:31]: Wrong slug :)

File: .github/workflows/deploy-plugin.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2.4.0
+
+    - name: WordPress.org Plugin Deploy
+      uses: 10up/action-wordpress-plugin-deploy@2.0.0
+      env:
+        SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+        SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+        SLUG: wpt-customm-mo-file
```
Comment (fxbenard) [inline .github/workflows/deploy-plugin.yml:22]: Wrong slug :)

File: .github/workflows/deploy-asset-readme-update.yml (hunk 1 above)
Comment (pedro-mendonca) [reply .github/workflows/deploy-asset-readme-update.yml]: haha! :D
Fixed!

File: .github/workflows/deploy-plugin.yml (hunk 2 above)
Comment (pedro-mendonca) [reply .github/workflows/deploy-plugin.yml]: Thanks! :D

**Pull request merged**